src/classes/ERR_Notifier.cls

```diff
@@ -105,179 +156,211 @@ public class ERR_Notifier {
+            for (Error__c error : errors) {
                 error.Email_Sent__c = true;
-
+            }
             update errors;
         }
     }
 
-    /*******************************************************************************************************
+    /**********************************************************************************************
     * @description Creates the email message to send as error notification.
     * @param context The context that triggered the error notification.
     * @param errorList The list of errors to include in the email.
-    * @param sendList The list of email recipients.
+    * @param recipientsList The list of email recipients.
     * @return 
     */
-    private static Messaging.SingleEmailMessage createEmailMessage(String context, List<Error__c> errorList, List<String> sendList) {
+    @TestVisible
+    private Messaging.SingleEmailMessage createEmailMessage(String context, List<Error__c> errorList, List<String> recipientsList) {
+        Error_Settings__c errorSettings = UTIL_CustomSettingsFacade.getErrorSettings();
+        String errorNotifRecipient = errorSettings.Error_Notifications_To__c;
+
         Messaging.SingleEmailMessage mail = new Messaging.SingleEmailMessage();
         mail.setUseSignature(false);
-        mail.setReplyTo('donotreply@salesforce.org');
-        mail.setSenderDisplayName('Nonprofit Success Pack');
-        mail.setSubject('Salesforce Error');
-        string bodyString = 'Organization: ' + UserInfo.getOrganizationName() + '(' + UserInfo.getOrganizationId() + ')\n';
-        bodyString += 'User: ' + UserInfo.getUserName() + '(' + UserInfo.getUserId() + ')\n\n';
-        bodyString += Label.ErrorEmailMessage + '\n';
-        if (context == ERR_Handler_API.Context.RD.name())
-            bodyString += system.label.npe03.RecurringDonationErrorEmailBody;
+        mail.setReplyTo(EMAIL_REPLY_TO);
+        mail.setSenderDisplayName(EMAIL_SENDER_NAME);
+        mail.setSubject(EMAIL_SUBJECT);
+
+        String bodyStringTemplate = 'Organization: {0} ({1})\nUser: {2} ({3})\n\n';
+        String bodyString = String.format(bodyStringTemplate, new List<String>{
+                UserInfo.getOrganizationName(), UserInfo.getOrganizationId(), UserInfo.getUserName(), UserInfo.getUserId()
+        });
+
+        bodyString += System.Label.ErrorEmailMessage + '\n';
+        if (context == ERR_Handler_API.Context.RD.name()) {
+            bodyString += System.Label.npe03.RecurringDonationErrorEmailBody;
+        }
+
         bodyString += '\n\nErrors:';
         Integer i = 1;
-        for (Error__c error : errorList){
-            bodystring += '\n\n----------\n Error #' + i + ': \n' + getErrorMessageBody(error);
+        for (Error__c error : errorList) {
+            bodyString += '\n\n----------\n Error #' + i + ': \n' + buildErrorMessageBody(error);
             i++;
         }
+
+        if (errorNotifRecipient != ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS) {
```

Comment:
@force2b Would this still add this message if errorNotifRecipient was a user ID and the user was active was active?

Reply:
Yes. The call to this method traces back to line 96 where `getErrorEmailRecipients()` is called to retrieve either a single email address (if the user is active) or a list of all admins.

Reply:
Unless I'm mistaken, wouldn't that mean we'd have to again check here if `errorNotifRecipient` is an active user? Otherwise this note would be added to the email even if the recipient is already active.

Reply:
Great catch @seanpat09. Just committed a fix.